### ISSUE TITLE
question: API to get download url for a file within shared_link folder

### ISSUE THREAD

**Issue by yarikoptic** [dropbox/dropbox-sdk-python#163]
Using `.files_list_folder(path="", shared_link="https://www.dropbox.com/sh/ofdd6ia32eo4wk5/AAA8553of0M2tQVlfR0S2GTFa")` it is possible to obtain a `ListFolderResult` instance with `FileMetadata` (and `FolderMetadata` for recursion) entries.  
For "owned" directories (not shared ones) it is possible to use `.sharing_create_shared_link(path, ...)` to obtain download URLs for individual files in your own dropbox `path`.
But I found nowhere in API a method to obtain a (public) download url (if files under `shared_link` were all shared as well) to the files discovered in the `shared_link` folder, e.g. as identified by `FileMetadata.rev` or `FileMetadata.id`.
Is there an API to achieve that?

Thank you in advance for the clarification/guidance!

**Comment by greg-db** [comment]
No, unfortunately it's not possible to get the shared link for a file inside a folder from a shared link for the folder. I'll pass this along as a feature request. I can't promise if or when that might be implemented though.

**Comment by yarikoptic** [comment]
Thank you in advance!
